CHANGES.rst

```diff
@@ -28,9 +28,11 @@ Improvements
 - Add button to adapt column widths in paper & contribution lists (:pr:`5642`)
 - Add event language settings to set default and additional languages (:issue:`5606`,
   :pr:`5607`, thanks :user:`vasantvohra`)
+- Hide the top box with the latest files/paper until it has been accepted (:issue:`5660`,
+  :pr:`5665`)
```

Comment:
Move this to the end of this changelog block (ie below the reminders thing, and ty for fixing that typo!)